src/controls/dynamicForm/DynamicForm.tsx

```diff
@@ -147,7 +147,12 @@ export class DynamicForm extends React.Component<
    * Default React component render method
    */
   public render(): JSX.Element {
-    const { customFormatting, fieldCollection, hiddenByFormula, infoErrorMessages, isSaving } = this.state;
+    let {fieldCollection} = this.state
+    const { customFormatting, hiddenByFormula, infoErrorMessages, isSaving } = this.state;
+
+    if (this.props.fieldOrder) {
```

Comment:
It would be better to check the length of the array (as it could be empty if controlled outside for example) to avoid unnecessary Map set.